tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -60,7 +60,14 @@ RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,
     # Use redis as a django cache https://pypi.org/project/django-redis/
     django-redis==5.2.0 \
     # uwsgi server https://pypi.org/project/uWSGI/
-    uwsgi==2.0.21
+    uwsgi==2.0.21 \
+    # Open edX Atlas tranlsation management tool https://pypi.org/project/openedx-atlas/
+    openedx-atlas==0.4.4
+
+# Update i18n: the equivalent of "make pull_translations"
+RUN find course_discovery/conf/locale -type d -mindepth 1 -maxdepth 1 -exec rm -rf {} + # Remove stale translation files
```

Comment:
This causes all translation files currently in course_discovery/conf/locale to be deleted. Not all translation files will be replaced by `atlas pull`. This means that some languages will no longer be available to existing users, which is not acceptable.

Is there an alternate solution? Can we just skip the deletion step?

Reply:
> This ideally shouldn't be a concern as long as both projects translations have the same completeness. Which is still work in progress.

OK. So this PR is not 100% ready to be merged, right?

> What if some translations are broken in the original repo? Then atlas pull won't fix those errors. What do you think?

I'm fine with existing bugs, I just don't want to create new ones. It's not atlas' job to fix existing translations issues.

> In MFEs, it's preferable to have only the used languages to reduce the JavaScript bundle because translations are compiled and bundled into the code. But this isn't an MFE.

That's a good point. But then again, it's not like MFE bundles are suddenly going to get very big. They already contain all the translation strings, right? Plus, it seems to me that translations strings take little size compared to vendor libraries.

Reply:
> That's a good point. But then again, it's not like MFE bundles are suddenly going to get very big. They already contain all the translation strings, right? Plus, it seems to me that translations strings take little size compared to vendor libraries.

This is still an open discussion in the Transifex Working Group. But yeah, it shouldn't be a concern in this context.

Reply:
@regisb. It's ready'ish, so no it's not ready. I just saw your comment 😿 

I'll check with the team on the plugin decision and get back to you. Thanks.

Reply:
I'm eager to see Atlas succeed and don't want to cause extra delays. If it's a chicken and egg thing, and building a plugin would further delay the project, then we can help resolve the situation by hiding the feature behind an undocumented feature flag in the tutor-discovery plugin, just like you suggested. We should make it clear that it's a temporary, unsupported feature flag, and the flag should not be present in other core plugins.

Thus I no longer have any objection to your PR and I'd say I'm ready to merge it when you are.

Reply:
To be clear, making it a plugin isn't a bad choice, in fact it's pretty good one. However, it's bit more costly than direct pull requests, so I'd like to hear from @brian-smith-tcril.

I imagine if we have an `tutor-atlas` plugin which adds Tutor patches to all other plugins would be a nice way to make atlas optional.

> the feature behind an undocumented feature flag in the tutor-discovery plugin, just like you suggested

Great, thanks for being open for the lesser option for the sake of timeline. Today, we'll know which way to go.